docs/update/dev.rst

```diff
@@ -98,40 +63,40 @@ Using the AUR (for Arch Linux users)
 With makepkg
 ~~~~~~~~~~~~
 
-::
+.. code-block:: bash
 
-    $ wget https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=python-pyfunceble-dev
-    $ makepkg
-    $ sudo pacman -U python-pyfunceble-dev*.tar.xz
+   wget https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=python-pyfunceble-dev
+   makepkg
+   sudo pacman -U python-pyfunceble-dev*.tar.xz
 
 With your favorite AUR helper
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. warning::
-    We do not recommend any AUR helper but keep in mind that some AUR helpers
-    are "better" than other.
-    For more information about your current (or any other) AUR helper please
-    report to `the ArchWiki page`_.
+   We do not recommend any AUR helper but keep in mind that some AUR helpers
+   are "better" than other.
+   For more information about your current (or any other) AUR helper please
+   report to `the ArchWiki page`_.
 
-::
+.. code-block:: bash
 
-    $ yourFavoriteAurHelper -Syu python-pyfunceble-dev
+   yourFavoriteAurHelper -Syu python-pyfunceble-dev
 
 Using docker (hub)
 """"""""""""""""""
 
 The image description can be found at https://hub.docker.com/r/pyfunceble/pyfunceble-dev
 
-::
+.. code-block:: bash
 
-   $ docker pull pyfunceble/pyfunceble-dev
+   docker pull pyfunceble/pyfunceble-dev
 
 Using :code:`conda`
```

Comment:
Can you please delete this section (`conda`) ? I dropped the automatic deployment of the conda package.

Reply:
Is it because you can't count to 2 and 3 things :rofl: 

Nope, I can re-write it to how to install conda and test within :open_mouth:   create a [issue](https://github.com/spirillen/PyFunceble/issues), and I'll fix it in a later release.

And bby that intergrate <https://github.com/funilrys/PyFunceble/issues/39> and link to <https://github.com/PyFunceble-Templates/pyfunceble-miniconda> for examples :smile: 

But that is a bigger (time) job, so I will prefer to leave it to later

Reply:
PS: It was actually a good thing you mentioned it, I had completely forgotten #39

Reply:
Actually, no. This section describes how to install the conda packages of PyFunceble (In the past: `conda install -c conda-forge -c pyfunceble pyfunceble-dev`) which I dropped with the release of `b1`. 

Out here it has nothing to do with the usage under/with conda as per https://github.com/PyFunceble-Templates/pyfunceble-miniconda. The https://github.com/PyFunceble-Templates/pyfunceble-miniconda thing should be indeed documented but not here. It should be a new subsection of the usage chapter. Not in the installation descriptions.

I don't want to give too much on the installation page, because it may become overwhelming to get the https://github.com/PyFunceble-Templates/pyfunceble-miniconda thing in there. Imagine the people who read about PyFunceble in a forum and just want to try it to see what it does. You don't want to overwhelm such people with a script explaining how to **use** (again _usage_ not _installation_) pyfunceble under an isolated Conda environment.

Out here it's just about the installation options.

Reply:
Well, at first I would move it to a more suitable space :smile: It just always nice to have some "drafts" as initiators.

Ok, I'll just kill the section.